app/resources/start.html.template

```diff
@@ -44,6 +56,7 @@
       {{env.start_page_custom_html|default:""|safe}}
     </div>
 
+    {% if not env.amp %}
```

Comment:
Maybe add a TODO comment and/or file an issue to support AMP share buttons?:
https://www.ampproject.org/docs/reference/components/amp-social-share

Reply:
done.